contracts/main/stablecoin-core/StableSwapModule.sol

```diff
@@ -20,7 +20,6 @@ contract StableSwapModule is PausableUpgradeable, ReentrancyGuardUpgradeable, IS
     using SafeToken for address;
 
     uint256 public constant ONE_DAY = 86400;
-    uint256 public constant MINIMUM_DAILY_SWAP_LIMIT = 1000 * 1e18;
```

Comment:
This could break Upgrading of contract. Not 100% sure as it is a constant and constants are part of bytecode, not of storage. Need to check. Better to not do this fix.

Reply:
thank you for pointing this out, but constants are not using any storage space, constant values replace all the usages during the compilation. you can read more here: https://docs.soliditylang.org/en/v0.8.20/contracts.html#constant-and-immutable-state-variables